presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/TableScanMatcher.java

```diff
@@ -107,4 +92,43 @@ public String toString()
                 .add("expectedOriginalConstraint", expectedOriginalConstraint.orElse(null))
                 .toString();
     }
+
+    /**
+     * By default, matches any TableScanNode.  Users add additional constraints by
+     * calling the various member functions of the Builder, typically named according
+     * to the field names of TableScanNode.
+     */
+    public static class Builder
```

Comment:
OK; I knew this was kinda weird looking but I just was following how `WindowMatcher` did it. I can add a commit fixing that too.